component/loki/process/stages/json.go

```diff
@@ -0,0 +1,180 @@
+package stages
+
+// This package is ported over from grafana/loki/clients/pkg/logentry/stages.
+// We aim to port the stages in steps, to avoid introducing huge amounts of
+// new code without being able to slowly review, examine and test them.
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	"github.com/jmespath/go-jmespath"
+	json "github.com/json-iterator/go"
+	"github.com/pkg/errors"
```

Comment:
Can you change any usage of `github.com/pkg/errors` to the stdlib errors package?